azclishell/app.py

```diff
@@ -393,7 +393,7 @@ def _special_cases(self, text, cmd, outside):
         if self.default_command:
             cmd = self.default_command + " " + cmd
 
-        if text.strip() == "quit" or text.strip() == "exit":
+        if cmd.strip() == "quit" or cmd.strip() == "exit":
```

Comment:
So it seems to me that as I start typing... text and cmd would be the same... and then once the shells basically switches over to parameter completions, then cmd is fixed even though text grows... is it something like that or not even?